supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/HtCondorWrapper.scala

```diff
@@ -153,4 +153,5 @@ object HtCondorRuntimeKeys {
   val Disk = "request_disk"
   val LogXml = "log_xml"
   val LeaveInQueue = "leave_in_queue"
+  val InitialWorkingDir = "Iwd"
```

Comment:
All these keys are defined by HtCondor scheduler. I cannot change this. Iwd param is one way to define initial working directory in HtCondor scheduler.